badge/main.py

```diff
@@ -83,8 +84,9 @@ def main():
         # Print the MAC ID on the console
         print('You are %s' % mac_id)
 
+        encoded_ap_neighbors = ujson.dumps(ap_neighbors).encode('utf8')
         try:
-          response = urequests.post(config.UPLOAD_URL + mac_id, json=ap_neighbors,  headers={'Content-Type': 'application/json', 'User-Agent': 'WarBadge Experimental ShmooCon 2018'})
```

Comment:
Can you please test the following?

`response = urequests.post(config.UPLOAD_URL + mac_id, json=ap_neighbors, ensure_ascii=False, headers={'Content-Type': 'application/json', 'User-Agent': 'WarBadge Experimental ShmooCon 2018'})`

Reply:
This does not look like it is posting successfully.  Adding some details to the try/except, looks like it is "unexpected keyword argument `ensure_ascii`".  Digging into it, the `ujson` library doesn't have the parameter `ensure_ascii`. The library `urequests` it is using `usjon` to do the dump but doesn't call encode, if it did, the the json parameter would work just fine.  It appears the key difference here is the `.encode('utf8')`.